operators/pkg/controller/common/annotation/controller_version.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package annotation
+
+import (
+	"github.com/elastic/cloud-on-k8s/operators/pkg/utils/k8s"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// ControllerVersionAnnotation is the annotation name that indicates the last controller version to update a resource
+const ControllerVersionAnnotation = "k8s.elastic.co/controller-version"
```

Comment:
I think we have `common.k8s.elastic.co` used in other places, maybe use it here as well?

Reply:
I'm good with that